model/resource/k8s.yaml

```diff
@@ -100,3 +100,12 @@ groups:
     attributes:
       - ref: k8s.cronjob.uid
       - ref: k8s.cronjob.name
+
+  - id: k8s.volume
```

Comment:
Let's avoid adding anything to resource until Entities SiG makes more progress.

I'd love to do more here, but right now Identifying + Descriptive attributes are blended in Resource and this is problematic for our long term success.

Let's remove the resource part, but we can reserve the other attributes with the meaning you suggest for general usage in OTEL.

Reply:
@jsuereth does this mean the semantic conventions are not accepting any more resource attribute proposals until Entities is stabilized?

Reply:
@jsuereth based on the discussion around resource attributes during today's SIG meeting, I believe at least `k8s.volume.name` meets the criteria of "not changing over the lifetime of the object".  I believe that `k8s.volume.type` also meets the criteria, but @ChrsMark can you confirm that a volume's type cant change without recreating?

Reply:
From https://kubernetes.io/docs/concepts/workloads/pods/#pod-update-and-replacement:

> Pod updates may not change fields other than spec.containers[*].image, spec.initContainers[*].image, spec.activeDeadlineSeconds or spec.tolerations. For spec.tolerations, you can only add new entries.

However thinking of it again, my concern is if we should make these resource attributes or metric attributes.
In `kubeletstats` we are going to use them as metric attributes right (for the `k8s.volume.*` metrics) ?

It seems we are in need of metric attributes here. So it sounds correct if we only keep them in the registry for now with a purpose similar to the [`system.device`](https://github.com/open-telemetry/semantic-conventions/blob/v1.26.0/docs/attributes-registry/system.md?plain=1#L24).

Reply:
See: https://github.com/open-telemetry/oteps/pull/264 for the path forward I'm proposing here.  Once we've agreed to what we're doing, we can unlock moving these attributes into Resource/Entity sections in Semconv.

Reply:
Thank's for the head's up!